### PR TITLE
Issue133/regexrepr

### LINKED ISSUE #239: Fix ControlLoop Model for SpecProduce case
The model allows non-unique hide actions, which TorXakis cannot deal with. 

Disabled testing SpecProduce example in #237; we need to fix the model and re-enable the test. Or get rid of it altogether, if it doesn't add any value as a TorXakis modelling example.

Also see #217 

On the other hand, this model can be used as an example after #222 is done. So we may not want to fix this at all...

### PR DESCRIPTION
**Pull request by pjljvandelaar**
one tick mark in #133
fixes #239

### CODE REVIEW

**Comment by pjljvandelaar** [comment]
solving code climate issues

had some issue with merging ;-(

File: sys/lexregexxsd/lexregexxsd.cabal [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  hs-source-dirs:       src
+  exposed-modules:      RegexAlex
+  
+  ghc-options:         -Werror -O2 -optc-O3 -optc-ffast-math
+  
+  build-tools:          alex
+  
+  build-depends:        array
+                      , base
+                      , text
```
Comment (capitanbatata) [inline sys/lexregexxsd/lexregexxsd.cabal]: Shouldn’t we add tests, specially when we’re developing new modules...

Comment (pjljvandelaar) [reply sys/lexregexxsd/lexregexxsd.cabal]: not a new content, just a new structure
but you are right we should test more!

Comment (capitanbatata) [reply sys/lexregexxsd/lexregexxsd.cabal]: Oh, I see. But probably we should strive to add tests when we touch some untested functionality, to a reasonable extent of course.

**Comment by capitanbatata** [comment]
Build is failing with error Progress: 78/80                   Not in scope: ‘txsCommandsFile

**Comment by pjljvandelaar** [comment]
indeed: in the comment some changes where made.. but not all needed...
See my latest push!

**Comment by capitanbatata** [comment]
Ok, I got scared for a bit...

**Comment by pjljvandelaar** [comment]
locally everything is fine:
Finished in 663.1347 seconds
45 examples, 0 failures

so now waiting on semaphoreci....

**Pull request merged**